src/sphinxcontrib/towncrier/__init__.py

```diff
@@ -85,6 +85,19 @@ def _get_changelog_draft_entries(
     return towncrier_output
 
 
+def _get_towncrier_config(
+    project_path: Path,
+    final_config_path: Path,
+) -> Dict[str, Any]:
```

Comment:
I won't request you to improve the typing but let's add a fixme note for future
```suggestion
        project_path: Path,
        final_config_path: Path,
) -> Dict[str, Any]:  # FIXME: add a better type
```